ddpui/schemas/chart_schema.py

```
@@ -90,6 +97,32 @@ class ChartDataPayload(Schema):
+    # explicit values are never overridden.
+    _EXTRA_CONFIG_MAP = {
+        "x_axis_column": "x_axis",
+        "y_axis_column": "y_axis",
+        "dimension_column": "dimension_col",
+        "extra_dimension_column": "extra_dimension",
+        "dimension_columns": "dimensions",
+        "metrics": "metrics",
+        "geographic_column": "geographic_column",
+        "value_column": "value_column",
+        "selected_geojson_id": "selected_geojson_id",
+    }
```

Ruff (0.15.7)

[warning] Mutable default value for class attribute

(RUF012)